games/u.yaml

```diff
@@ -160,6 +160,35 @@
   type: remake
   updated: 2015-04-05
   url: http://jackyjjc.com/
+  
+- name: UnCiv
+  type: remake
+  originals:
+  - Civilization V
+  repo: https://github.com/yairm210/UnCiv
+  url: https://play.google.com/store/apps/details?id=com.unciv.app
+  development: active
+  status: semi-playable
```

Comment:
Not sure what the standard is, so for the record, it's fully playable on its own but doesn't have all the features of the original